Use proper logging mechanism instead of notify on macOS (#535)
conda/constructor: CHANGELOG.rst

.. current developments

XXXX-XX-XX   X.X.X:
===================

Enhancements:
-------------

> * The graphical macOS installers now use the macOS logging system to log

Author: ```suggestion
* macOS installers now use the macOS logging system to log
```

Enhancements:
-------------

> * PKG installers now default to the macOS logging system to log messages.
>   UI notifications can be enabled with `progress_notifications`
>   (off by default). (#535)
> 

Reviewer: ```suggestion
```

Enhancements:
-------------

> * macOS installers now use the macOS logging system to log
>   messages. Previously, notification badges would be displayed. (#535)

Reviewer: ```suggestion
* PKG installers now default to the macOS logging system to log messages.
  UI notifications can be enabled with `progress_notifications`
  (off by default). (#535)
```
